packages/vaadin-grid/test/keyboard-navigation.test.js

```diff
@@ -1,13 +1,16 @@
 import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
-import { aTimeout, fixtureSync, nextFrame } from '@open-wc/testing-helpers/index-no-side-effects.js';
 import {
+  aTimeout,
+  down as mouseDown,
```

Comment:
It might be worth to consider renaming `down` to `mouseDown` and `up` to `mouseUp` as well.

Reply:
Thanks. There are some more helpers in grid tests, and I'm going to handle those in a separate PR.